styles/style.css

```diff
@@ -0,0 +1,448 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto');
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:300');
+
```

Comment:
Please document which parts of the CSS represent what. Checkout [Resume Review](https://github.com/mihi-r/resume-review-day/blob/master/styles/main.css) for an example.